### PR TITLE
Bump SDK dependencies

### PR DESCRIPTION
**Pull request by hrls**
Solves https://github.com/Atleta-network/atleta/issues/117

### CODE REVIEW

File: node/src/eth.rs [hunk 1]
```diff
@@ -172,10 +173,10 @@ pub async fn spawn_frontier_tasks<RuntimeApi, Executor>(
                 fc_mapping_sync::sql::SyncWorker::run(
                     client.clone(),
                     backend,
-                    Arc::new(b),
+                    b.clone(),
                     client.import_notification_stream(),
                     fc_mapping_sync::sql::SyncWorkerConfig {
-                        read_notification_timeout: Duration::from_secs(10),
+                        read_notification_timeout: Duration::from_secs(30), // TODO: tbd
```
Comment (ales-tsurko) [inline node/src/eth.rs]: Why’s this changed?

File: scripts/localnet [hunk 2]
```diff
@@ -37,6 +37,7 @@ bob() {
         --tmp \
         --bob \
         --port 30334 \
+        --unsafe-force-node-key-generation \
```
Comment (ales-tsurko) [inline scripts/localnet:40]: Will the update also affect how we run the network? Should we specify this flag for our nodes?

File: node/src/eth.rs (hunk 1 above)
Comment (hrls) [reply node/src/eth.rs]: Just synced with value from template. Looks like we can keep our old 10 secs.

File: scripts/localnet (hunk 2 above)
Comment (hrls) [reply scripts/localnet:40]: It allows node to generate on run any _node key_ when the expected one is not found.
This _cli key_ can be omitted if the _node key_ is generated explicitly at some step. So I think, no, we don't need this in our nodes.

Comment (ales-tsurko) [reply scripts/localnet:40]: Do we need it if we didn’t set the node key with a flag (explicitly)?

File: node/src/eth.rs (hunk 1 above)
Comment (ales-tsurko) [reply node/src/eth.rs]: Yeah, let’s keep the old value

File: scripts/localnet (hunk 2 above)
Comment (hrls) [reply scripts/localnet:40]: I think yes, if key doesn't exist. 
https://github.com/paritytech/polkadot-sdk/blob/8c8edacf8942298c3807a2e192860da9e7e4996a/substrate/client/cli/src/params/node_key_params.rs#L118

Comment (ales-tsurko) [reply scripts/localnet:40]: Then we need to update our deploy scripts as well with either setting the ids or adding this flag, as in this case the change will break them. Could you open an issue and mark it as blocking for this PR? Needs a little investigation how much it’s “unsafe” and whether we need to generate the ids.

Comment (hrls) [reply scripts/localnet:40]: Here the issue #120 
I'm not sure if GitHub has any way to mark a PR as blocked by an issue (other than just comment).

**Comment by hrls** [comment]
Blocked by #120

**Review by ales-tsurko — DISMISSED** [review]

**Comment by ales-tsurko** [comment]
don't forget to merge it with changes from #130 first (just a reminder).

**Review by DmitriVT — DISMISSED** [review]

**Review by DmitriVT — APPROVED** [review]

**Pull request merged**